app/lib/bk/compat/parsers/bitrise/steps.rb

```diff
@@ -15,11 +15,42 @@ def translator(type, inputs)
           send("translate_#{type.downcase.gsub('-', '_')}", inputs)
         end
 
+        def translate_brew_install(inputs)
+          return '# Invalid brew-install configuration!' unless inputs.include?('verbose_log')
+
+          generate_brew_install_command(inputs)
+        end
+
+        def generate_brew_install_command(inputs)
+          if inputs.fetch('packages', false)
+            install_type = inputs.fetch('upgrade', false) ? 'reinstall' : 'install'
+            "brew #{install_type}#{inputs['verbose_log'] ? ' -vd' : nil} #{inputs['packages']}"
+          elsif inputs.fetch('use_brewfile', false) && inputs.fetch('brewfile_path', false)
```

Comment:
This is the same as:

```suggestion
          if inputs['packages']
            install_type = inputs.fetch('upgrade', false) ? 'reinstall' : 'install'
            "brew #{install_type}#{inputs['verbose_log'] ? ' -vd' : nil} #{inputs['packages']}"
          elsif inputs['use_brewfile'] && inputs['brewfile_path']
```

Reply:
Adjusted back 👍